### DISCUSSION TITLE
Version 1.71 re-enabled Ethernet port

### DISCUSSION THREAD

**Discussion by bartashevich** [nathanmarlor/foxess_modbus#589] [Q&A]
Just upgraded to latest firmware 1.71 and ethernet port seem to work with this HA modbus addon.

Before I know it used to freeze inverter after certain amount of time.

I'll keep it running to see if there is improvement and no longer freezes.

**Comment by paultweedy77** [comment]
Well, that's unexpected to say the least.. very interested to know if it seems any more stable than before!
(Although academic to me personally now as I have Gen2 inverter that has no ethernet port..)

**Comment by bartashevich** [comment]
3 days later inverter froze as in the past. So nothing is improved. Continues to be very bad method of retrieving info. Unless the issue somehow is related to this addon.
